test/pie-minus-value.html

```diff
@@ -0,0 +1,105 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <script src="lib/simpleRequire.js"></script>
+        <script src="lib/config.js"></script>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+    </head>
+    <body>
+        <style>
+            h1 {
+                line-height: 60px;
+                height: 60px;
+                background: #146402;
+                text-align: center;
+                font-weight: bold;
+                color: #eee;
+                font-size: 14px;
+            }
+            .chart {
+                height: 600px;
+            }
+        </style>
+
+        <div id="main" class="chart"></div>
+        <script>
+            require(["echarts"], function (echarts) {
+                var chart = echarts.init(document.getElementById("main"));
+
+                chart.setOption({
+                    aria: {
+                        enabled: true,
+                    },
+                    title: {
+                        text: `The minus value slice should render like 0 value,
+                        while the tooltip should show the origin minus value`,
+                        x: "center",
+                    },
+                    tooltip: {
+                        trigger: "item",
+                        formatter: "{a} <br/>{b} : {c} ({d}%)",
+                    },
+                    legend: {
+                        orient: "vertical",
+                        left: "left",
+                        data: [
+                            "直接访问",
+                            "邮件营销",
+                            "联盟广告",
+                            "视频广告(value is 0)",
+                            "搜索引擎(value is minus)",
+                        ],
+                    },
+                    series: [
+                        {
+                            name: "访问来源",
+                            type: "pie",
+                            radius: "55%",
+                            center: ["50%", "60%"],
+                            selectedMode: "single",
+                            data: [
+                                { value: 335, name: "直接访问" },
```

Comment:
As @plainheart suggested, I think we should add a test case that all data are negative.

Reply:
When all the data are negative, the pie chart is empty.
I'm not sure if this is the expected result 🐒

![image](https://user-images.githubusercontent.com/10973821/121203407-a6986c00-c8a8-11eb-8be2-80fb7db10fda.png)

Reply:
> When all the data are negative, the pie chart is empty.
> I'm not sure if this is the expected result 🐒

At least, it's unexpected for me. This will break the previous behavior. And I think it's meaningful when all value is negative. In that case, considering them as `0` may be not a good way.

Reply:
By default (if [series-pie.stillShowZeroSum](https://echarts.apache.org/zh/option.html#series-pie.stillShowZeroSum) is not set to be `false`), when all data is negative, it should behave the same as when all data is zero.

![image](https://user-images.githubusercontent.com/779050/121656021-a5965300-cad1-11eb-8ce0-344e4cb982dc.png)

Reply:
If the canvas is blank, it looks like there is an error to the users. So perhaps we could display an empty light gray circle when no data (sum is zero or all is `'-'` or negative). We should also provide options to configure the color.
The legend color doesn't look right to me. They should have different colors for different series.

Reply:
@Ovilia this sounds reasonable, I'll give it a try~
>  display an empty light gray circle when no data

For the color issue, all three legends' are in blue because they are all filtered by `negativeDataFilter`. And the filtered legend's default color is blue